src/views/developers/developers.jsx

```diff
@@ -157,6 +162,12 @@ var Developers = React.createClass({
                         </FlexRow>
                     </section>
 
+                    <section id="join">
+                        <span className="nav-spacer"></span>
+                        <h2><FormattedMessage id='developers.joinTitle' /></h2>
+                        <p><FormattedMessage id='developers.joinBody' /></p>
```

Comment:
@TheGrits This looks great. One minor issue. The developers.joinBody needs to be included with a FormattedHTMLMessage instead of FormattedMessage because the text includes HTML tags. Other than that, it looks good.